SETUP/API.md

```diff
@@ -31,12 +35,18 @@ RewriteRule ^/api/(.*)$ %{DOCUMENT_ROOT}/c/api/index.php?url=$1 [L,QSA]
 
 ## Access Control
 
-API access is limited to individuals with API keys. API keys are associated
-with users and the API calls are restricted to the same information as the
-user who owns it.
+API access is limited to individuals with API keys or valid PHP session keys.
```

Comment:
You think there is value in allowing us to disable api access from api keys while retaining php session keys? This would allow us to disable it say if some security issue is discovered with the key access or some pgdp implementation decides they don't want it?

Reply:
I had considered that and decided that there probably isn't value in splitting them apart. If a pgdp deployment doesn't want to enable session keys they can simply not give any out (since they are manually opt-in) or empty any that have been. Since the API codepath is identical for the two authentication methods, if there is a security issue found the site will want to disable both of them (particularly since all logged-in users have defacto access to the API with the right HTTP request and session cookies).